src/main/java/org/craftercms/studio/impl/v1/service/content/ContentServiceImpl.java

```diff
@@ -2109,6 +2109,8 @@ public boolean revertContentItem(@ValidateStringParam String site,
             if (isEmpty(commitId)) {
                 return false;
             }
+            String username = securityService.getCurrentUser();
```

Comment:
Is this related to this issue/PR?

Reply:
@sumerjabri no, it is not. I will exclude from this one